features/individual-transforms.yml

```diff
@@ -2,3 +2,10 @@ name: Individual transform properties
 description: Transform elements with separate `translate`, `rotate`, and `scale` CSS properties.
```

Comment:
non-blocking suggestion: The description could be more explicit about how these allow individual transformation vs the grouped property, giving context to why the feature is named as such. Eg, I like the "simple transforms independently" in the spec language.